Source/Csla.Shared/Configuration/CslaConfigurationOptions.cs

```diff
@@ -23,7 +25,7 @@ public class CslaConfigurationOptions
     public string PropertyChangedMode
     {
       get { return ConfigurationManager.AppSettings["CslaPropertyChangedMode"]; }
-      set { ConfigurationManager.AppSettings["CslaPropertyChangedMode"] = value; }
+      set { ApplicationContext.PropertyChangedMode = (PropertyChangedModes)Enum.Parse(typeof(PropertyChangedModes), value); }
```

Comment:
I think that in this (and the other properties) we should also be setting the AppSettings[] value as you were before? Keep AppSettings up to date with the current configuration values?